docs/reference/module-api/module-overview.md

```diff
@@ -720,6 +741,25 @@ module.exports = {
 };
 ```
 
+If including a route option, use an object. See the [route options](#route-options) section for more.
+
+Each route function takes the Express arguments `req` (the [request object](https://expressjs.com/en/api.html#req)) and `res` (the [response object](https://expressjs.com/en/api.html#res)). The functions must generate a response via `res` to avoid leaking resources, typically using the `res.redirect` or `res.send` methods.
```

Comment:
From here down, until the `### Route options`, in this green section didn't change.